packages/nimble-blazor/Tests/NimbleBlazor.Tests.Acceptance/Tests/ThemeProviderTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Playwright;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Acceptance
+{
+    public class ThemeProviderTests : AcceptanceTestsBase
+    {
+        public ThemeProviderTests(PlaywrightFixture playwrightFixture, BlazorServerWebHostFixture blazorServerClassFixture)
+            : base(playwrightFixture, blazorServerClassFixture)
+        {
+        }
+
+        [Fact]
+        public async Task ThemeProvider_ValidityAndCheckValidityWorkAsync()
+        {
+            await using (var pageWrapper = await NewPageForRouteAsync("ThemeProvider"))
+            {
+                var page = pageWrapper.Page;
+                var validButton = page.Locator("nimble-button", new PageLocatorOptions() { HasText = "Set Valid Lang" });
+                var invalidButton = page.Locator("nimble-button", new PageLocatorOptions() { HasText = "Set Invalid Lang" });
+                var isValidCheckbox = page.Locator("nimble-checkbox", new PageLocatorOptions() { HasText = "IsValid" });
+                var langIsInvalidCheckbox = page.Locator("nimble-checkbox", new PageLocatorOptions() { HasText = "InvalidLang" });
+                await Assertions.Expect(validButton).ToBeVisibleAsync();
```

Comment:
Why do we need this assertion that the button is visible? From what I can tell, that isn't adding a lot of value to this test.

Reply:
Was thinking we should make sure the button was there before trying to click it, but I think `ClickAsync()` is probably already handling that.